### ISSUE TITLE
[SCSS Transition] Height - Width - Font-size

### ISSUE THREAD

**Issue by Mbk10201** [Facepunch/sbox-issues#5328]
### For?

S&Box

### What can't you do?

There is not transition applied yet on the font-size, height, width, etc.. 
I guess it hasn't been implemented yet.

### How would you like it to work?

To be able to transit them alone or by all 

transition: font-size 0.5s ease;
transition: height 2s ease-in;
transition: all 0.5s ease;
etc...

### What have you tried?

Nothing yet

**Comment by Mbk10201** [comment]
> Yeah a shit ton of transitions are missing. I remember not being able to use border-color.

yeah that too, and many more others but they will probably add them soon.

**Comment by xezno** [comment]
All the examples you provided work fine on my end, I need more info (ideally a repro project)

https://github.com/Facepunch/sbox-issues/assets/12881812/e5bee002-8626-44da-89b7-9754ac4e101d
